docs/checkers/index.md

```diff
@@ -488,7 +513,7 @@ This error occurs when a non-mapping value is used in a place where mapping is e
 
 ### Unnecessary negation (C0117)
 
-This error occurs when a boolean expression contains an nnecessary negation. If we are getting this
+This error occurs when a boolean expression contains an unnecessary negation. If we are getting this
```

Comment:
This change is not related to this pull request, and so should not be included here.
 
However, this is a good catch, so what you could do it make a second, separate pull request that just fixes this issue!

Reply:
It was clear, I just haven't had the change to do so. Now it is fixed and a separate pull request has been created.